python/base_computer/ship_view.py

```diff
@@ -56,9 +69,18 @@ def get_int(ship_stats, key):
     return int(ship_stats[key])
 
 def get_dbl(ship_stats, key, divider = 1.0):
-    fl = float(ship_stats[key])/divider
-    #s = f"{key} {fl}"
-    #print(s)
+    try:
+        fl = float(ship_stats[key])/divider
+    except KeyError:
+        print('KeyError in get_dbl: ', key, ' not found in ship_stats')
```

Comment:
FYI - since we're using Py3 we can use[ f-Strings](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings) here we makes formatting really nice and easy:
```python
print(f'KeyError in get_dbl: {key} not found in ship_stats')
```

Reply:
Should it be f"blurb" and not f'blurb'?

Reply:
@royfalk it doesn't really matter which; typically you use one if you want to use the other:
```python
f"blurb 'b' "
f'blurb "b" '
```
they both denote a string.